pylint_plugins.py

```diff
@@ -90,13 +99,17 @@ def fake_class(name_or_class_obj, members=()):
     'ipalib.config.Env': [
         {'__d': ['get']},
         {'__done': ['add']},
-        'xmlrpc_uri',
```

Comment:
Is there a reason these members were here before? Why are they being removed? The commit message does not explain that.